wordpress-mobile/WordPress-iOS
# Issue/7068 dynamic type on reader and post list

**Fixes** #7068 

This PR provides support for Dynamic Type in the following places:
- Reader Post List
- Reader Detail
- Blog Post List

More PRs to come once this one is approved and merged to add support in other areas of the app.

**To test:**
Enable "Larger Text" on Settings->General->Accessibility->Larger Text
Check that the Reader Post List cells have bigger text and look good.
Check that the Reader Detail view has bigger text and looks good.
Check that the Blog Post List cells have bigger text and look good.

Since some font sizes changed slightly, I am adding before and after pictures for @mattmiklic to verify that everything looks good.

| Develop        | This branch           |
| ------------- |-------------|
|![postlist](https://cloud.githubusercontent.com/assets/5558824/26259039/9b215ed6-3c9d-11e7-866f-ca06e5431152.png)| ![postlistcurrent](https://cloud.githubusercontent.com/assets/5558824/26259041/9b587fc4-3c9d-11e7-8b24-04bf542ba6f6.png)|
![readerdetail](https://cloud.githubusercontent.com/assets/5558824/26259040/9b57c25a-3c9d-11e7-8755-c46b64ccdd48.png)|![readerdetailcurrent](https://cloud.githubusercontent.com/assets/5558824/26259043/9b5adfd0-3c9d-11e7-9c67-05286d1ec126.png)
![readerlist](https://cloud.githubusercontent.com/assets/5558824/26259042/9b58cfec-3c9d-11e7-97a5-169ac9d447b9.png) | ![readerlistcurrent](https://cloud.githubusercontent.com/assets/5558824/26259044/9b5b08d4-3c9d-11e7-96be-ecb17bb83f03.png)|


Needs review: @nheagy
